config/reports.json

```diff
@@ -428,6 +430,132 @@
 			"timeseries": {
 				"enabled": false
 			}
+		},
+		"workboxAdoption": {
+			"name": "Workbox Adoption",
+			"type": "URLs",
+			"downIsBad": true,
+			"description": "[Workbox](https://developers.google.com/web/tools/workbox/) is a set of libraries and Node.js modules that make it easy to cache assets and take full advantage of features used to build Progressive Web Apps. This report shows the number of PWA URLs that use them.",
+			"histogram": {
+				"minDate": "2016_01_01"
+			}
+		},
+		"medianLighthouseScore": {
+			"name": "Lighthouse Scores",
+			"type": "PWA Score",
+			"downIsBad": true,
+			"description": "[Lighthouse](https://developers.google.com/web/tools/lighthouse/) is an open-source, automated tool for improving the quality of web pages. This report shows the average Lighthouse PWA score.",
+			"histogram": {
+				"minDate": "2017_06_01"
+			}
+		},
+		"lighthousePWAs": {
+			"name": "PWAs Tested with Lighthouse",
+			"type": "URLs",
+			"downIsBad": true,
+			"description": "[Lighthouse](https://developers.google.com/web/tools/lighthouse/) is an open-source, automated tool for improving the quality of web pages. This report shows how many PWAs have been tested with the tool and have achieved a PWA score of 75 and higher.",
+			"histogram": {
+				"minDate": "2017_06_01"
+			}
+		},
+		"usecounterPWAs": {
+			"name": "Pages Controlled by a Service Worker",
+			"type": "URLs",
+			"downIsBad": true,
+			"description": "The ```ServiceWorkerControlledPage``` [use counters](https://cs.chromium.org/chromium/src/third_party/blink/public/platform/web_feature.mojom) triggers whenever a page is controlled by a service worker. This report shows how many PWAs have triggered this use counter.",
+			"histogram": {
+				"minDate": "2016_12_01"
+			}
+		},
+		"responseBodiesPWAs": {
+			"name": "Pages with Service Worker and Manifest",
+			"type": "URLs",
+			"downIsBad": true,
+			"description": "This report shows how many pages register a Service Worker and have a Web App Manifest.",
+			"histogram": {
+				"minDate": "2016_01_01"
+			}
+		},
+		"stateOfTheUnionPWAs": {
+			"name": "All known PWAs",
+			"type": "URLs",
+			"downIsBad": true,
+			"description": "This report shows how many pages either register a Service Worker and have a Web App Manifest, trigger the ```ServiceWorkerControlledPage``` use counter, or were successfully tested with Lighthouse.",
+			"histogram": {
+				"minDate": "2017_06_01"
+			}
+		},
+		"installEvents": {
+			"name": "\"Install\" Events",
```

Comment:
I don't think we currently support markdown in metric names, but would you prefer to have this appear as "`install` Events" with code formatting? Also note that I think it might be better to show the event name in lowercase as it'd be written in actual code.

Reply:
Addressed in https://github.com/HTTPArchive/httparchive.org/pull/51/commits/f1d09dea78bea2c48035ed315219740657f4d7a6.